diff --git a/hatch_build.py b/hatch_build.py
index 036079a..f629435 100644
--- a/hatch_build.py
+++ b/hatch_build.py
@@ -76,10 +76,9 @@ class BuildHook(BuildHookInterface):
             )
 
             # Compile an executable
-            subprocess.check_call((
-                sys.executable, "-m", "ninja",
-                "-C", Dirs.build,
-            ))
+            subprocess.check_call(
+                ("ninja", "-C", Dirs.build)
+            )
 
             # Find and vendor the binary for this version
             binary = Dirs.build.joinpath("nvibrant")
